Wrappers/Python/ccpi/viewer/CILViewer2D.py

```diff
@@ -545,7 +547,10 @@ def OnKeyPress(self, interactor, event):
             self.SetEventActive('UPDATE_WINDOW_LEVEL_UNDER_CURSOR')
         elif interactor.GetKeyCode() == "t":
             # tracing event is captured by widget
-            pass
+            if (self._viewer.imageTracer.GetEnabled()):
```

Comment:
Changed it so we enable and disable the tracer ourselves by interpreting the key press, instead of the tracer doing this